### PR TITLE
Default tip label

### PR DESCRIPTION
**Pull request by joverlee521**
## Description of proposed changes

Testing <https://github.com/nextstrain/auspice/pull/1668>

Following changes made in mpox
<https://github.com/nextstrain/mpox/commit/4a268babe797bc4361444fc7cf2ae434129d1da2>

## Checklist

- [x] Checks pass
- [x] [Test run](https://github.com/nextstrain/zika/actions/runs/10565516945)

### CODE REVIEW

File: phylogenetic/defaults/auspice_config.json [hunk 1]
```diff
@@ -1,56 +1,60 @@
...
+      "key": "region",
+      "title": "region",
+      "type": "categorical"
+    }
+  ],
+  "geo_resolutions": [
+    "country",
+    "region"
+  ],
+  "display_defaults": {
```
Comment (jameshadfield) [inline phylogenetic/defaults/auspice_config.json:48]: This commit changed this file from CRLF to LF, so GitHub shows every line as changed. The salient changes are:

```diff
   "display_defaults": {
-    "map_triplicate": true
+    "map_triplicate": true,
+    "tip_label": "strain"
   },
+  "metadata_columns": [
+    "strain"
+  ],
   "filters": [
```

Comment (joverlee521) [reply phylogenetic/defaults/auspice_config.json:48]: Yup, also viewable in the [specific commit changes](https://github.com/nextstrain/zika/pull/72/commits/8fb92b53fbd6382b5b4dadaec72d861f9698947a#diff-eec2a07074cc3f264719563e7deca28eea78cc40a224230b135f075f23a687d0)

Comment (jameshadfield) [reply phylogenetic/defaults/auspice_config.json:48]: Yeah - totally missed that & was editing my comment when you replied!

**Comment by jameshadfield** [comment]
Testing URLs:
* [current nextstrain](https://next.nextstrain.org/staging/zika/trials/default-tip-label/zika) - console error "Invalid JSON-defined tip label: strain" and we fall back to using the node.name (accession)
* [nextstrain using preview version of Auspice](https://nextstrain-s-nextstrain-lteeid.herokuapp.com/staging/zika/trials/default-tip-label/zika) - tips are labelled by strain as expected

**Comment by joverlee521** [comment]
> @joverlee521 any reason not to un-draft this and merge it?

Nope, just cleaned up the config in https://github.com/nextstrain/zika/pull/72/commits/21473b9ce587ce939d0cef43f26c2a9f8aa6be83.

**Pull request merged**